### DISCUSSION TITLE
Problema con branch

### DISCUSSION THREAD

**Discussion by Agusbruzzese** [INGCOM-UNRN-P1/cursada-2023#47] [Preguntas y respuestas] [answered]
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/96895097/3ffc33e7-39ca-4c9d-80bd-e924c6d84aad)
Ayuda

**Comment by martinvilu** [comment, marked as answer]
Lo ideal es trabajar en un solo lugar, para evitar esto.

Primero, `git pull` y fijate si git puede resolver el problema por su cuenta.

**Reply by Agusbruzzese** [reply]
Probe varioas pulls y nada por ahora

**Reply by martinvilu** [reply]
Hay cambios en el mismo archivo, en tu computadora y en lo que está en GitHub.

El mensaje que publicas, se resuelve con `git pull` porque el `push` fue rechazado porque lo que estás subiendo está “atrás” con respecto al repositorio en GitHub.

Ahora que has hecho `pull`, ¿que dice `git status`?

**Reply by martinvilu** [reply]
¿Y?
